frontend/src/utils/constants.tsx

```diff
@@ -234,14 +240,18 @@ export const polls: Array<SelectablePoll> = [
     defaultAnonEntityActions: [AnalysisPageLink],
     defaultRecoLanguageDiscovery: true,
     defaultRecoSearchParams: 'date=Month',
-    displayOrder: 10,
     mainCriterionName: 'largely_recommended',
+    displayOrder: 20,
```

Comment:
This parameter of the Poll is used to sort the list of polls in the PollSelector (top left menu on the header) and the PollList (home page). Putting it at 20 makes it equal to the value for the poll `presidentielle2022` but we prefer that the poll video always appears first.

```suggestion
    displayOrder: 10,
```